yt_dlp/extractor/nhk.py

```diff
@@ -782,7 +863,7 @@ def _real_extract(self, url):
         site_id, corner_id, headline_id = self._match_valid_url(url).group('site', 'corner', 'headline')
         programme_id = f'{site_id}_{corner_id}'
 
-        if site_id == 'F261':  # XXX: News programmes use old API (for now?)
+        if site_id == '18439M2W42':  # XXX: News programmes use old API (for now?)
```

Comment:
```suggestion
        # XXX: News programmes use the old API
        # Can't move this to NhkRadioNewsPageIE because news items still use the normal URL format
        if site_id == '18439M2W42':
```